ui/web/src/services/RestServices.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Pull from master and use services/RestServices/index.js file

Reply:
Fixed